catalog.html

```diff
@@ -181,34 +181,34 @@ <h3 class="hotel-name">GreenTree Inn</h3>
           </ul>
           <ul class="pagination">
             <li class="pagination-item">
-              <a class="pagination-link-pagination-current-first">1</a>
+              <a class="pagination-link-pagination-current">1</a>
             </li>
             <li class="pagination-item">
-              <a class="pagination-link-pagination-current">2</a>
+              <a class="pagination-link-pagination">2</a>
```

Comment:
```suggestion
              <a class="pagination-link">2</a>
```
И у аналогичных. 

Зачем нам так увеличивать название класса? Не похоже, что это может дать нам больше понимания.